src/main/drivers/pwm_output.h

```diff
@@ -165,10 +167,18 @@ void servoDevInit(const servoDevConfig_t *servoDevConfig);
 void pwmServoConfig(const struct timerHardware_s *timerHardware, uint8_t servoIndex, uint16_t servoPwmRate, uint16_t servoCenterPulse);
 
 #ifdef USE_DSHOT
+typedef uint8_t(*loadDmaBufferFuncPtr)(motorDmaOutput_t *const motor, uint16_t packet);  // function pointer used to encode a digital motor value into the DMA buffer representation
```

Comment:
Personally I prefer 
```
typedef uint8_t loadDmaBufferFuncPtr)(motorDmaOutput_t *const motor, uint16_t packet);
```
In C declaration, `*` denotes pointer and is easier to read than Hungarian notation